ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -384,17 +391,31 @@ objects:
         #!/bin/bash
         set -euo pipefail
 
-        trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit /tmp/setup-failed; fi; exit "${rc}"' EXIT
+        trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/shared/setup-success; else touch /tmp/shared/exit /tmp/shared/setup-failed; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+        if [[ "${CLUSTER_TYPE}" != "gcp" ]] && [[ "${CLUSTER_ENABLE_NESTED_VIRT-false}" == "true" ]]; then
+            echo "CLUSTER_ENABLE_NESTED_VIRT is unsupported for cluster type '${CLUSTER_TYPE}'"
+            exit 1
+        fi
+
+        export PATH=/tmp/shared:$PATH  # gain access to oc
+        while ! command -V oc; do sleep 1; done # poll to make sure that the test container has dropped oc into the shared volume
+
+        # we need jq for later machine rescaling
+        (cd /tmp/shared && curl -L -o jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 && chmod +x jq)
+        if ! command -V jq; then
+            echo "Failed to fetch jq"
+            exit 1
+        fi
+
         cp "$(command -v openshift-install)" /tmp
```

Comment:
@dhiller this needs to be `cp "$(command -v openshift-install)" /tmp/shared` now for the teardown container

Reply:
or, change the volume back to `/tmp` and then change the `jq` install to `/tmp` instead of `/tmp/shared`  - this will be the least change